specificatie/domain.yaml

```diff
@@ -241,6 +307,73 @@ components:
           $ref: "#/components/schemas/Naam"
+    Hypotheek:
+      type: "object"
+      description: |
+        Bij een hypotheek op de kadastraal onroerende zaak dient het eigendomsrecht van de hypotheekgever als onderpand voor een geldlening of krediet bij de hypotheekhouder (geldverstrekker).
+
+        Waardelijst in deze component :
+        - [omschrijvingBetrokkenRecht](http://www.kadaster.nl/schemas/waardelijsten/BetrokkenRecht/)
+      properties:
+        identificatie:
+          type: "string"
+        domein:
+          type: "string"
+          description: |
+                        Het domein waartoe de identificatie behoort.
+        aandeelInBetrokkenRecht:
+          $ref: "#/components/schemas/TypeBreuk"
+        gedeeltelijkeBezwaringOudObject:
+          type: "boolean"
+        omschrijvingBetrokkenRecht:
+          $ref: "#/components/schemas/Waardelijst"
+          description: |
+                        Mogelijke waarden zijn te vinden in deze [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/BetrokkenRecht/)
+        toelichtingBewaarder:
+          type: "string"
+          description: |
+                        Toelichtende tekst van de bewaarder. De bewaarder is iemand die bij het Kadaster werkt. Hij schrijft stukken in in de openbare registers en de basisregistratie Kadaster conform de Kadasterwet.
+        betreftGedeelteVanPerceel:
+          type: "boolean"
+        aantekeningen:
+          type: array
+          items:
+            $ref: "#/components/schemas/AantekeningBasis"
+            description: |
+                          Een aantekening is een verwijzing naar een ter inschrijving aangeboden stuk. Een aantekening op een hypotheek is bijvoorbeeld een rangwisseling hypotheek, waarbij een hypotheek voorrang heeft boven eerder ingeschreven hypotheken of rechten op dezelfde kadastraal onroerende zaak.
+        hypotheekhouders:
+          description: |
+                        Een hypotheekhouder vestigt als geldverstrekker een recht van hypotheek als zekerheid voor de lening.
+          type: array
+          items:
+            $ref: "#/components/schemas/PersoonBeperkt"
+            description: |
+                          Geldverstrekker die een recht van hypotheek vestigt als zekerheid voor de lening.
+        bedragZekerheidsstelling:
+          $ref: "#/components/schemas/Bedrag"
+          description: |
+                        Het bedrag waarvoor hypothecaire zekerheid is gegeven door onderpandgever en waarvoor het recht van Hypotheek is gevestigd.
+        stukIdentificaties:
+          type: "array"
+          description: |
+                        Identificatie van het stuk. Een stuk is een brondocument dat aanleiding geeft
+                        tot een wijziging van de gegevens in een basisregistratie.
+                        Dit kan een aangeboden stuk of een kadasterstuk zijn.
+          items:
+            type: string
+        omschrijvingGekozenWoonplaats:
```

Comment:
Waarom is deze toegevoegd? Het is mij uit de naam niet duidelijk wat dit gegeven betekent. Dan zou ik dus een toelichting (description) verwachten. 

Ik zie dit gegeven trouwens ook niet in het imKad staan.

Dit gegeven wordt nu nog niet geleverd in de implementatie. Is er een issue toegewezen aan @rhengeveld en @kad-hebbim om dit toe te voegen in de implementatie?

Reply:
omschrijvingGekozenWoonplaats zit in het IMKAD in het stukdeel. We hebben in een overleg besloten deze property bij de hypotheek op te nemen omdat deze property alleen daar van toepassing is. 

Ik worstel een beetje met wanneer ik nu wel of niet een description moet toevoegen. De omschrijvingGekozenWoonplaats is voor mij volstrekt vanzelfsprekend in de context van een hypotheek. Blijkbaar ben ik niet representatief en kan ik me niet voldoende inleven in de developer die hiermee aan de slag gaat.